### ISSUE TITLE
Homepage: Align the order of items/cta links in Home page Benefit and News promo blocks by unlinking news promo title

### ISSUE THREAD

**Issue by jilladams** [department-of-veterans-affairs/va.gov-cms#12098]
## Description 

Homepage mid-point review 508 feedback: 
 Harmonize the benefit and new promo heading/link/description pattern to be a consistent interaction.  

### Home page benefit promo
At the top of the page, the PACT Act section follows the pattern:

> [heading]
> [description]
> [link]

### Home page news promo
Further down the page, the Pathfinder news item follows the pattern:

> [linked heading]
> [description]
> [link]

The PACT Act section (benefit promo) is the preferred pattern because it 
- keeps the purpose of the heading separate from the link 
- "Learn what the PACT Act means to you" link is a single link with unambiguous text, which is a smoother interaction for screen reader users.

## Task
- [ ] Review with @thejordanwood for design feedback in refinement

## Acceptance Criteria
- [ ] News promo is presented as unlinked text

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [X] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by jilladams** [comment]
This task was identified in the homepage midpoint review, and initially wrapped into #11392 . I've broken it out into its own piece of work for the sake of easier execution / tracking.

**Comment by jilladams** [comment]
Per review with Dave today: we could verify this with metrics potentially, to measure interactions with both of these elements to align on whichever one is consistently driven by veteran interaction. Mainly re: whether people _try_ to click the headers.

**Comment by laflannery** [comment]
Just noting that [CAIA would like to move away from linked headings](https://dsva.slack.com/archives/C023ZKDJJK1/p1688595889940609) - this guidance isn't in any way formalized and I don't know yet when it will be but unlinking the new headline on the homepage would fit with this direction

**Comment by sky3an1co** [comment]
Released Feb 5, 2026, Sprint 13: https://github.com/department-of-veterans-affairs/content-build/releases/tag/content-preview%2Fv0.0.3868

Prod example: https://www.va.gov/#va------------news
